packages/forms/src/UIForm/fields/Datalist/__snapshots__/Datalist.component.test.js.snap

```diff
@@ -27,7 +27,7 @@ exports[`Datalist component should render 1`] = `
     onFocus={[Function]}
     placeholder="Type here"
     readOnly={false}
-    restricted={false}
+    restricted={true}
```

Comment:
seems like the default value assigned in `get(this.props, 'schema.restricted', false)` is not taken into account, right ?

Reply:
I think you might look at the wrong place, both schema and schemaMultiSection do have `restricted: true`

Reply:
oh you're right, my bad !